src/routes/safe/store/actions/transactionsNew/setPreviousPage.ts

```diff
@@ -0,0 +1,5 @@
+import { createAction } from 'redux-actions'
+
+export const SET_PREVIOUS_PAGE = 'SET_PREVIOUS_PAGE'
+
+export const setPreviousPage = createAction(SET_PREVIOUS_PAGE)
```

Comment:
not sure about the rules to defining actions, but... could be a possibility to have only 1 file (e.g "pagination.ts") where we can export the 3 actions:
~setNextPage~ nextPage
~addNewTransaction~ getMore / loadMore
~setPreviosPage~ previousPage

Reply:
done